proto/FeatureCollection/FeatureCollection.proto

```diff
@@ -86,11 +88,11 @@ message FeatureCollectionPBuffer {
   }
 
   message Field {
-    string name = 1;
+    string name = 1; 

```

Comment:
You accidentally checked in some trailing whitespace here, and below around L144 + L175